File: atm_money_counting.py
money = 1000
total = 0
card = "harsha"
password = 1234
count = 3

for x in range(count):
    print("no. of atm counts -", x)
    if card == "harsha" and password == 1234:
        while total < money:
            total = total + 100
            print("Money counting-", total)

        print("Total amount:", total)
        total = 0  
    else:
        print("Card or password incorrect!")



Atm_card = "card"
pin = 1234
Money = 1000
Temp = 0
outer = 1  


if (Atm_card == "card") and (pin == 1234):
    if outer == 1:
        while Temp <= Money:
            print("count:", Temp, "Total Amount:", Money)
            Temp += 100
        print("Take your money:", Money, "rupee")
    else:
        print("Only one attempt is possible")
elif (Atm_card == "card") or (pin == 1234):
    print("Any one value is wrong")
else:
    print("Both values are wrong")

print("Thank you for using the ATM")

File: atm_money_counting.c
#include <stdio.h>
#include <string.h>

int main() {
    int money = 1000;
    int total = 0;
    char card[] = "harsha";
    char entered_card[20];
    int password = 1234;
    int entered_pin;
    int count = 3;

    printf("SecureATM\n");

    for (int x = 0; x < count; x++) {
        printf("\nNo. of ATM counts - %d\n", x);

        strcpy(entered_card, "harsha");
        entered_pin = 1234;

        if (strcmp(card, entered_card) == 0 && entered_pin == password) {
            while (total < money) {
                total += 100;
                printf("Money counting - %d\n", total);
            }
            printf("Total amount: %d\n", total);
            total = 0;
        } else {
            printf("Card or password incorrect!\n");
        }
    }


    char Atm_card[] = "card";
    char input_card[20];
    int pin = 1234;
    int input_pin;
    int Money = 1000;
    int Temp = 0;
    int outer = 1;

    strcpy(input_card, "card");
    input_pin = 1234;

    if (strcmp(input_card, Atm_card) == 0 && input_pin == pin) {
        if (outer == 1) {
            while (Temp <= Money) {
                printf("Count: %d  Total Amount: %d\n", Temp, Money);
                Temp += 100;
            }
            printf("Take your money: %d rupee\n", Money);
        } else {
            printf("Only one attempt is possible\n");
        }
    } else if (strcmp(input_card, Atm_card) == 0 || input_pin == pin) {
        printf("Any one value is wrong\n");
    } else {
        printf("Both values are wrong\n");
    }

    printf("\nThank you for using SecureATM C\n");

    return 0;
}
